server/api/utils/oauth_helper.py

```diff
@@ -0,0 +1,85 @@
+import base64
```

Comment:
### Nitpick

If we're making use of `PEP8` standard then your imports should be arranged according to the rules [here](https://www.python.org/dev/peps/pep-0008/?#imports).

cc @mbithenzomo